qulice-pmd/src/main/java/com/qulice/pmd/rules/ProhibitOldStyleAssertionsRule.java

```diff
@@ -0,0 +1,198 @@
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.qulice.pmd.rules;
+
+import net.sourceforge.pmd.lang.ast.Node;
+import net.sourceforge.pmd.lang.java.ast.ASTImportDeclaration;
+import net.sourceforge.pmd.lang.java.ast.ASTMethodDeclaration;
+import net.sourceforge.pmd.lang.java.ast.ASTName;
+import net.sourceforge.pmd.lang.java.ast.ASTPrimaryExpression;
+import net.sourceforge.pmd.lang.java.ast.ASTPrimaryPrefix;
+import net.sourceforge.pmd.lang.java.ast.ASTStatementExpression;
+import net.sourceforge.pmd.lang.java.rule.junit.AbstractJUnitRule;
+
+/**
+ * Rule to check old style assertions in JUnit tests.
+ * @author Viktor Kuchyn (kuchin.victor@gmail.com)
+ * @version $Id$
+ */
+public final class ProhibitOldStyleAssertionsRule extends AbstractJUnitRule {
+
+    /**
+     * Mask of prohibited imports.
+     */
+    private static final String[] PROHIBITED = {
+        "org.junit.Assert.assert",
+        "junit.framework.Assert.assert",
+    };
+
+    @Override
+    public Object visit(final ASTMethodDeclaration method, final Object data) {
+        if (this.isJUnitMethod(method, data)
+            && this.containsOldStyleAssert(method.getBlock())) {
+            this.addViolation(data, method);
+        }
+        return data;
+    }
+
+    @Override
+    public Object visit(final ASTImportDeclaration imp, final Object data) {
+        if (ProhibitOldStyleAssertionsRule.isArrayMatchesValue(
+            imp.getImportedName(), ProhibitOldStyleAssertionsRule.PROHIBITED
+        )) {
+            this.addViolation(data, imp);
+        }
+        return super.visit(imp, data);
+    }
+
+    /**
+     * Checks if string matches at least one element in array.
+     * @param value String value to be matched
+     * @param array Array of templates
+     * @return False if no matches found, true if at least one
+     */
+    private static boolean isArrayMatchesValue(final String value,
+        final String... array) {
+        boolean matches = false;
+        for (final String element : array) {
+            if (value.contains(element)) {
+                matches = true;
+                break;
+            }
+        }
+        return matches;
+    }
+
+    /**
+     * Recursively verifies if node contains old style assert statements.
+     * @param node Root statement node to search
+     * @return True if statement contains old style assertions, false otherwise
+     */
+    private boolean containsOldStyleAssert(
+        final Node node
```

Comment:
@vkuchyn We can move this parameter (and the paren+curly brace below) up to line 99.